android/app/src/main/AndroidManifest.xml

```diff
@@ -773,6 +773,7 @@
     <activity
         android:name="app.organicmaps.MwmActivity"
         android:launchMode="singleTask"
+        android:configChanges="uiMode"
```

Comment:
It will receive all ui mode changes:
https://developer.android.com/guide/topics/manifest/activity-element#config

> A change to the user interface mode, such as when the user places the device into a desk or car dock, or the night mode changes. For more information about the different UI modes, see [UiModeManager](https://developer.android.com/reference/android/app/UiModeManager).